src/world/area_hos/hos_04/A274B0.c

```diff
@@ -10,7 +10,9 @@ ApiStatus N(SetCamXXVfov)(Evt* script, s32 isInitialCall) {
 
 INCLUDE_ASM(s32, "world/area_hos/hos_04/A274B0", func_80240654_A27524);
 
-INCLUDE_ASM(s32, "world/area_hos/hos_04/A274B0", func_8024068C_A2755C);
+ApiStatus func_8024068C_A2755C(void) {
```

Comment:
All these script functions which return ApiStatus should take as their arguments `Evt* script, s32 isInitialCall` - applies below as well